MAVProxy/modules/mavproxy_steamvr.py

```diff
@@ -0,0 +1,303 @@
+"""
+use SteamVR tracker data to provide VISION_POSITION_ESTIMATE and GPS_INPUT data
+"""
```

Comment:
In the header comments: add in any extra packages required for this module (ie openvr) and links to any special install/setup instructions